test/hotspot/jtreg/runtime/appcds/MoveJDKTest.java

```diff
@@ -27,12 +27,10 @@
  * @summary Test that CDS still works when the JDK is moved to a new directory
  * @bug 8272345
  * @requires vm.cds
+ * @requires vm.flagless
  * @comment This test doesn't work on Windows because it depends on symlinks
  * @requires os.family != "windows"
  * @library /test/lib
- * @modules java.base/jdk.internal.misc
- *          java.management
- *          jdk.jartool/sun.tools.jar
  * @compile test-classes/Hello.java
```

Comment:
Includes the changes from [JDK-8229267](https://bugs.openjdk.org/browse/JDK-8229267) ( commit [e9b271d](https://github.com/openjdk/jdk/commit/e9b271dd1065994e4538e3a628e3f3542fdcacb5) ) on this file